tests/wp-tests/BM/Metabox/Posts/TagtocategoryTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+	
+	/**
+	 * Test BM_Metabox_Posts_TagToCategory class.
+	 */
+	class TagtocategoryTest extends BM_TestCase {
```

Comment:
Check the latest code in `dev/2.0.0` branch. You have to extend a different class.

Reply:
I did this last Friday.